birthdays-project/BirthdayApp/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class Birthdate(models.Model):
+    dob = models.DateField(max_length=8)
+
+    # Displays date in a dd-MMM-yy format.
+    def __str__(self):
+        return '{:%d %b %y}'.format(self.dob)
+
+
+# Creates a user Model.
+class User(models.Model):
+    firstName = models.TextField(max_length=20, verbose_name='First Name')
+    lastName = models.TextField(max_length=30, verbose_name='Last Name')
+    dob = models.ForeignKey(Birthdate, editable=True, max_length=8,
```

Comment:
Even though many people are born on the same day, I would not model that with a many-to-one relation. Please drop the Birthdate model and define the date_of_birth on this model directly. It simplifies model and code.